package.json

```diff
@@ -31,40 +32,30 @@
     "react-popper": "^2.3.0",
     "react-split": "^2.0.14",
     "stream-http": "^3.2.0",
+    "vite": "^4.5.0",
     "vscode-ws-jsonrpc": "^2.0.0",
     "ws": "^8.9.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.0",
-    "@babel/core": "^7.1.0",
-    "@babel/preset-env": "^7.1.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.18.6",
```

Comment:
was babel just needed for webpack?

Reply:
vite normally uses swi instead. I am not sure whether I set it up correctly, but it seems to work.